addons/website_event/models/event_tag.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class EventTagCategory(models.Model):
+    _name = 'event.tag'
+    _inherit = ['event.tag', 'website.published.mixin']
+    event_tag_website_id = fields.Many2one(
```

Comment:
Why not calling it "website_id" ? As generally we will do "tag.website_id", calling it "event_tag_website_id" is a bit verbose ("tag.event_tag_website_id" is too much tags IMHO).

Reply:
@tde-banana-odoo i don't know :D i mean `event_tag_website_id` seemed more informative from where that website id came from, i will change it to `website_id`

Reply:
@tde-banana-odoo i don't know :D i mean `event_tag_website_id` seemed more informative from where that website id came from, i will change it to `website_id`